docs/modules/ROOT/pages/vshn-managed/postgresql/major-upgrade.adoc

```diff
@@ -0,0 +1,53 @@
+= PostgreSQL Major Version Upgrade
+
+== Overview
+VSHNPostgreSQL supports major version upgrades up to the latest PostgreSQL 17.
+
+[WARNING]
+====
+Carefully follow the steps below to avoid prolonged downtime or data loss.
+====
+
+== Preparing for the Upgrade
+Major version upgrades carry risks, so before proceeding, ensure the following:
+
+. **Check Disk Space**: Although the upgrade process uses hard links for data migration, we recommend having at least double the free storage as a precaution.
+. **Verify Extensions**: If your instance uses extensions, confirm they are available and compatible with the target PostgreSQL version. Check the https://stackgres.io/extensions/[StackGres Extensions] repository for compatibility.
+. **Backup Your Data**: Always create a fresh backup before starting the upgrade.
+. **Review Configuration Changes**: If your current PostgreSQL configuration is incompatible with the new version, contact support before proceeding.
+. **Test the Upgrade**: Perform the upgrade in a test environment to validate the process before applying it to production.
+
+== Limitations
```

Comment:
Maybe add another limitation that progress currently can't be followed.

Reply:
yep